src/index.html

```diff
@@ -7,6 +7,308 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <div class="header__logo"></div>
```

Comment:
Logo is a meaningful part of websites. The <img> tag provides meaningful information about the content, which is especially useful for search engines and assistive technologies. Using `background-image` in CSS is a suitable method for adding decorative or non-essential images to your website.